core/audits/network-requests.js

```diff
@@ -77,6 +82,7 @@ class NetworkRequests extends Audit {
         priority: record.priority,
         isLinkPreload,
         experimentalFromMainFrame,
+        entity,
```

Comment:
Sorry, I was still kind of vague. Rather than the whole entity (which could potentially be repeated multiple times in this array), the idea (I believe from @alexnj) was

```suggestion
        entityName: entity.name,
```
which is unique and can be used for easy lookup in the `entites` array

Reply:
Thanks @brendankenny, still sketching it out 👍